1-exercises/G-array-methods/exercise.js

```diff
@@ -4,7 +4,7 @@
 */
 
 let numbers = [3, 2, 1];
-let sortedNumbers; // complete this statement
+let sortedNumbers= numbers.sort();// complete this statement
```

Comment:
`.sort()` mutates the array, so it's good practice to copy before sorting, for example:
` let sortedNumbers= [...numbers].sort();` you can learn more about the spread operator here https://www.javascripttutorial.net/es6/javascript-spread/